doc/invalidation-handlers.rst

```diff
@@ -9,9 +9,15 @@ to simplify common operations.
 Tag Handler
 -----------
 
-The tag handler helps you to invalidate all cache entries that where marked
-with a specified tag. It works only with a ``CacheInvalidator`` that supports
-``CacheInvalidator::INVALIDATE``.
+.. versionadded:: 1.3
```

Comment:
this does not render as nicely as expected, because our sphinx setup does not load basic.css but only theme.css - @ddeboer would you know why we do not load default.css and what i could do to get the

    .versionmodified {
        font-style: italic;
    }

into our css, or load basic.css?